x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/use_results_rollup/index.tsx

```diff
@@ -185,7 +185,6 @@ export const useResultsRollup = ({ ilmPhases, patterns }: Props): UseResultsRoll
         const indexId = getIndexId({ indexName, stats });
 
         if (
```

Comment:
since `indexId` was here and now we use `indexName` should   we replace indexId with indexName here as well?
```suggestion
        if ( indexName != null &
```

Reply:
indexName is always available on both serverless and the traditional Kibana, so we are fine not to check it here when collecting the tracking information. However we can remove checking `ilmExplain` here, as it's not available on serverless and we still want to track this behaviour.

<img width="2559" alt="Screenshot 2024-04-22 at 17 09 05" src="https://github.com/elastic/kibana/assets/6295984/559f1881-54da-4413-9735-525c1cd35558">